app/routes/cases/submissions/submission.js

```diff
@@ -130,6 +130,9 @@ export default class CasesSubmissionsSubmissionRoute extends Route {
   async afterModel(model) {
     const decisionmakingFlow = await model.belongsTo('decisionmakingFlow').reload();
     await decisionmakingFlow?.case;
+    if (this.currentSession.may('treat-and-accept-submissions')) {
+      await model.internalReview;
+    }
```

Comment:
I haven't done this in the agendaitem routes. Might be cleaner for views then to have a silent await in the template